src/components/DFViewer.tsx

```diff
@@ -22,6 +22,10 @@ const updateAtMatch = (
 export type setColumFunc = (newCol: string) => void;
 export type StyleAnalogue = Record<string, string | number>;
 
+
+/*
+
+ */
```

Comment:
Actually fairly happy with DFViewer,  I basically only have to edit this component to port from react-data-grid to ag-grid